WordPressAuthenticator/Unified Auth/View Related/2FA/TwoFAViewController.swift

```diff
@@ -51,6 +51,8 @@ final class TwoFAViewController: LoginViewController {
         
         if isMovingToParent {
             tracker.track(step: .twoFactorAuthentication)
+        } else {
+            tracker.set(step: .twoFactorAuthentication)
         }
```

Comment:
I think you can remove the if / else statement here and just use `tracker.set(step: .twoFactorAuthentication)`

Reply:
When entering the VC, we track the step:

`tracker.track(step: .twoFactorAuthentication)`

When showing this VC as a result of another child VC being popped, we're only setting the step back, without tracking anything:

`tracker.set(step: .twoFactorAuthentication)`

If I changed to the proposed code, we'd never track the step.

Reply:
ohhh I got you. I didn't expand the lines and didn't realize this was in the viewDidAppear.